.github/workflows/ci.yml

```diff
@@ -7,9 +7,13 @@ on:
       - '**.md'
     branches:
       - master
+      - 'release/**'
```

Comment:
Not really sure if we want to actually push images off of a release branch. This will result in images being called `...:release/v1.1` which could lead to people think that this is an actual release build which it isn't.
Can you follow my thoughts?

Reply:
Yes, makes sense, could bring the confusion